packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -451,12 +463,32 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     return html`
       <div class="root" dir=${this.direction}>
         ${navigationTemplate}
+        <div class="close-card-container">
+          <fluent-button appearance="lightweight" class="close-button" @click=${() => this.closeCard()}>
+              ${getSvg(SvgIcon.Close)}
+          </fluent-button>
+        </div>
         <div class="person-details-container">${personDetailsTemplate}</div>
         <div class="expanded-details-container">${expandedDetailsTemplate}</div>
       </div>
     `;
   }
 
+  /**
+   * Render the state when no data is available.
+   *
+   * @protected
+   * @returns {TemplateResult}
+   * @memberof MgtPersonCard
+   */
+  protected closeCard() {
+    const flyout = this.parentElement.parentElement as MgtFlyout;
```

Comment:
This won't work if interaction type is set to `person-card="click"`

Reply:
Thanks for finding this, solved with a little workaround: 
f5dec79